Installers/Linux/RUN/postinstall.sh

```diff
@@ -88,6 +84,18 @@ case "$choice2" in
 	sudo -H -u $SUDO_USER bash -c 'mdtool setup install $DIR/Main/MonoDevelop.MonoGame.mpack'
 esac
 
+#MonoGame.Content.Builder.targets
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame
+fi
+
+if [ ! -d /usr/lib/mono/xbuild/MonoGame/v3.0 ]; then
+	mkdir /usr/lib/mono/xbuild/MonoGame/v3.0
+fi
+
+sudo cp $DIR/Main/MonoGame.Content.Builder.targets tmp/usr/lib/mono/xbuild/MonoGame/v3.0/
```

Comment:
On windows they go in 
``
Program Files\MonoGame\v3.0\Assemblies\Linux
Program Files\MonoGame\v3.0\Assemblies\Windows8
```
etc

On Mac we install to 
```
$(MSBuildExtensionsPath)\MonoGame\v3.0\Assemblies\Linux
```

You can get the idea from https://github.com/mono/MonoGame/blob/develop/Installers/default.build#L64

So we probably want to follow a similar pattern as OS X. You probably want to install Linux mainly I guess, not sure about Android/iOS because they can't be used on Linux (or built).. same for the other platforms.